applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -170,6 +172,11 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	ret = data_fifo_pointer_first_vacant_get(&ble_fifo_rx, (void *)&iso_received, K_NO_WAIT);
 	ERR_CHK_MSG(ret, "Unable to get FIFO pointer");
 
+	if ((data_size != ENC_MAX_FRAME_SIZE) && (data_size != last_data_size)) {
+		LOG_WRN("Data length mismatch: %d (received), %d (codec)", data_size,
```

Comment:
Should be LOG_ERR

Reply:
I think LOG_WRN would be proper, since if we are using USB as audio interface, there will be lenght = 0 packets when there's no stream plays from PC.